src/UI/Carousel.module.less

```diff
@@ -0,0 +1,78 @@
+.carouselContainer {
```

Comment:
Wist niet dat we gewoon less konden gebruiken :/

Reply:
Kan ook nu pas, sinds https://github.com/parley-messaging/web-library/pull/62/commits/820a10acb935afee85ee40999818f48efccc12cf

Reply:
Nice, vind de gelaagdheid in less erg overzichtelijk.